_posts/2022-12-14-cockpit-282.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 282
+author: garrett
+date: '2022-12-14'
+tags: cockpit
+slug: cockpit-282
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 282:
+
+
+## Accounts: Redesign and include groups
+
+The Accounts page has been redesigned. Groups are listed at the top of the page and the list of users is enhanced with details and filtering. Additionally, a user's groups can be edited while viewing details for the user.
+
+![screenshot of redesign and include groups]({{ site.baseurl }}/images/282-accounts-redesign-and-include-groups.png)
+
+![screenshot of redesign and include groups]({{ site.baseurl }}/images/282-accounts-redesign-and-include-groups-2.png)
+
+## Right-to-left language support
+
+Right-to-left language support has been enabled for languages that require it. Our most complete RTL language is Hebrew. When choosing Hebrew in the languages list, the layout will shift and become right-aligned.
+
+In addition to basic RTL support, we have started to modify Cockpit and PatternFly to work with RTL languages in a better way. This is not complete, but we wanted to enable this as soon as possible, so people could use it and report bugs.
+
+We are aware that there are some basic issues across Cockpit. Most of Cockpit should be usable in RTL, but there are still some alignment and spacing problems.
+
+The most affected pages at the moment are:
+- the login page (which does not currently adapt to RTL)
```

Comment:
Not true. It works just fine (but you need to mount the static dir locally).

Reply:
Whoops! I forgot about the login screen being a special case. Thanks! Changed the intro based on this comment and our F2F discussion about it.